docs/services/vision/detection.md

```diff
@@ -24,8 +24,8 @@ The returned detections consist of the bounding box around the identified object
 
 You can use the following types of detectors:
 
-- [**color_detector**](#configure-a-color_detector): A heuristic detector that draws boxes around objects according to their hue (does not detect black, gray, and white).
-- [**mlmodel**](#configure-a-mlmodel-detector): A machine learning detector that draws bounding boxes according to a ML model available on the robot’s hard drive.
+- [**color detection**](#configure-a-color_detector): A heuristic detector that draws boxes around objects according to their hue (does not detect black, gray, and white).
+- [**object detection**](#configure-a-mlmodel-detector): A detector that draws bounding boxes according to an [ML model](/services/ml/) available on the robot’s hard drive.
```

Comment:
```suggestion
- [**ML model**](#configure-a-mlmodel-detector): A detector that draws bounding boxes according to an [ML model](/services/ml/) available on the robot’s hard drive.
```
[opt] Calling it object detection may confuse folks because you could also detect a colored object.

If you want it to be object detection, I think caps make sense in this list if you're going human readable here:
```suggestion
- [**Object detection**](#configure-a-mlmodel-detector): A detector that draws bounding boxes according to an [ML model](/services/ml/) available on the robot’s hard drive.
```

Reply:
This is feedback from product directly that how it currently is is confusing. I see what you mean but I think it's still better for understanding to call  it object detection - and the rest of the sentence should add more context